src/components/Section.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
In future avoid using external ui libraries. You'll learn more by implementing the ui yourself